pkgs/applications/misc/michabo/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, mkDerivation
+, writeText
+, fetchFromGitLab
+, qmake
+# qt
+, qtbase
+, qtwebsockets
+}:
+
+let
+  desktopFile = writeText "michabo.desktop" ''
```

Comment:
@fgaz this slipped by me first time around. Any reason not to use `makeDesktopItem` here instead?

Reply:
None other than me not knowing about it :)
Updated